branca/element.py

```diff
@@ -57,11 +57,13 @@ def __init__(self, template=None, template_name=None):
         self._env = ENV
         self._children = OrderedDict()
         self._parent = None
+        self.template = template
+        self.template_name = template_name
 
-        if template is not None:
-            self._template = Template(template)
-        elif template_name is not None:
-            self._template = ENV.get_template(template_name)
+        if self.template is not None:
+            self._template = Template(self.template)
+        elif self.template_name is not None:
+            self._template = ENV.get_template(self.template_name)
```

Comment:
Could you revert the changes to these lines? They are not really doing any meaningful change, so in that case I'd prefer to keep the diff smaller. Smaller diffs means less chances for mistakes.

Reply:
Done !